docs/environments.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Environments
+---
+
+# Environments
+
+## Descriptions
+<!--
+ from ticket #2516 "Our four environments are: Dev, Sandbox, Pre Prod, Prod"
+
+ Description of environments from Slack 
+ https://dsva.slack.com/archives/C06EB8S1FT9/p1710884055935049
+
+ https://adhoc.slack.com/archives/C042L8C962D/p1712157437377449?thread_ts=1712058858.556049&cid=C042L8C962D
+
+ VA Profile has an environment called INT. int currently connects to EB dev. INT will connect to EB sandbox.
+-->
+
+Event Bus [Kafka](https://kafka.apache.org/) and applications are deployed into several environments for different purposes. The environments are described below.
+
+- **`Dev`** is an environment intended for developer experiments with no expectations for stability. We try our best, but we might release broken code to this environment.
+- **`Sandbox`** is a stable environment that external teams can connect to. Applications deployed to this environment must have been approved by our cATO process.
+- **`Pre-prod`** is a testing environment that has been cleared for PHI and PII. `Pre-prod` may be used for performance tests.
+- **`Prod`** is our production environment.
+
+## Deployment Schedule
+
+We deploy any new releases to `sandbox` and `prod` on Wednesdays.
+
+## Properties
+
+| Environment<br>Name | AWS Account  | Kubernetes<br>Cluster | Kubernetes<br>Namespace     | PHI, PII<br>Allowed |
```

Comment:
Are the account and Kubernetes values needed? Based on these points from the ticket description, I think we can leave them off and just have name, description, and PHI/PII:

>* Defines all Event Bus code environments: what is the name of the environment and a concise description of what it is used for.
>* When we deploy to environment (Weds) (primarily sandbox/prod)
>* If environment is allowed to (or may) have PHI/PII
>* Don't make this complicated, it's just a quick reference page for us and client teams.

Reply:
I'm putting the AWS account and kubernetes values in because they are confusing and I don't see where that information is available all together. It would be improved if I could link to the definitive sources for the information, but in the case of the namespaces, I don't think we have access to that.

Reply:
Do producers/consumers need these values? If so I think we should keep them, but otherwise maybe they would be better to document internally for our team's use

Reply:
I don't think producers/consumers need this, but we can definitely document it internally somewhere else

Reply:
**Agreed.** The page is for producers, consumers and teams who might want to be one or the other. So any information that doesn't help them integrate, or decide to integrate, is surplus to requirements here. [Emily's summary](https://app.zenhub.com/workspaces/event-bus-team-632b50a9a7bf8400277f2cf8/issues/gh/department-of-veterans-affairs/ves/2516#:~:text=Slack%20post%20from%20Emily), which I did my best to capture in [this comparison chart,](https://app.zenhub.com/workspaces/event-bus-team-632b50a9a7bf8400277f2cf8/issues/gh/department-of-veterans-affairs/ves/2516#:~:text=Slack%20post%20from%20Emily) still seems like a good, approachable intro for people who aren't yet in the weeds of AWS Kafka.

Reply:
Having these values may help our team communicate what these environments are and what their requirements are to other teams. Different teams use different terminology, and there has been confusion when talking to other teams and just using environment names over what the requirements are for connecting, but other people could give details on that better than me.

Should there be a "properties" table, and what properties need to go in it if any?

Reply:
Would be good to get some other opinions on this as well, but I think the `PHI/PII allowed` property is very helpful. Other than that I don't think the AWS Account or Kubernetes info is very helpful in describing requirements to connect. I wonder if broker addresses would be more helpful, instead of AWS Account and Kubernetes info - that would let users know what addresses to use for connecting. I think the descriptions above this table are also helpful in describing the differences between these environments

Reply:
As discussed in today's standup and the engineering mob session, I've changed the environment table to have only environment Name, broker endpoints, and whether PHI and PII are allowed, and I've removed 'dev'.